cog/cog-client.cpp

```diff
@@ -103,6 +157,20 @@ cog_client_class_init (CogClientClass *klass)
 
   object_class->constructed = cog_client_constructed;
   object_class->finalize = cog_client_finalize;
+
+  object_class->set_property = cog_client_set_property;
+  object_class->get_property = cog_client_get_property;
+
+  g_object_class_install_property (object_class,
+                                   PROP_REGION,
+                                   g_param_spec_string ("region",
```

Comment:
I would either make this an enum property, with values corresponding to http://sdk.amazonaws.com/cpp/api/LATEST/namespace_aws_1_1_region.html, or keep it as a string property but define string constants (that can be introspected.)

Reply:
I've defined an enum to map to Aws::Region